contrib/juliac/juliac-buildscript.jl

```diff
@@ -73,6 +129,14 @@ let mod = Base.include(Main, ARGS[1])
     if ARGS[3] == "true"
         ccall(:jl_add_ccallable_entrypoints, Cvoid, ())
     end
+
+    # Export info about entrypoints and structs needed to create header files
+    if length(ARGS) >= 4
```

Comment:
Can we make this a `--export-abi build.json` (or other name - open to bikeshedding) instead of a positional arg?